setup.py

```diff
@@ -42,9 +42,11 @@
     include_package_data=True,
     extras_require={
         'docs': [
-            'sphinx',
+            'sphinx==4.0.0',
```

Comment:
We should keep an eye on this before merging! :slightly_smiling_face:

Reply:
Yes good flag, this was a temporary fix but yah we should check if their bug is better now 😄

Reply:
Removed the pin to `sphinx==4.0.0` here: https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3355/commits/1d3e5ac921bad27bc8f68906cf24bb951822bd22